javascript-source/core/commandCoolDown.js

```diff
@@ -31,19 +31,41 @@
 
     $.raffleCommand = null;
 
+    const   Type = {
+                User: 'user',
+                Global: 'global',
+            },
+            Operation = {
+                UnSet: -1,
+                UnChanged: 0,
```

Comment:
Codacy has a fix for the issue: [Delete `··········`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
      UnChanged: 0,
```